Fixed spacing and errors on the song list.

JackpotSHUnited opened the pull request:
New to github so there was some trial and/or error involved, but the song list should be perfectly fixed and up-to-date.

JackpotSHUnited committed:
Fixed spacing and fixed errors on song list.

JackpotSHUnited committed:
Fixed spacing and fixed errors on song list.

JackpotSHUnited committed:
Fixed spacing and fixed errors on song list.

bb010g commented:
Can you [squash the commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)?

JackpotSHUnited commented:
Does it have to be me that does it? I'm looking at that link and I just started drooling in stupidity.

JackpotSHUnited commented:
rebase

bb010g commented:
I can walk you through it on the Gitter.

somealternateaccount commented:
When you're squashing these commits please add a description to the final commit regarding what changes you made so the people of the future don't have to dig through a diff regarding what changes you made.

Azura01 commented:
Is this being worked on? Perhaps it would be easiest if Jackpot handed off the files for someone else to deal with?

JackpotSHUnited commented:
There are no files, it's just the TXT file that lists all the song changes. The spacing is currently out of whack, so I fixed up the spacing and corrected a few errors. 

Mushroom Kingdom II (Melee) should say Peach's Castle (Super Smash Bros.)

A second copy of Metal Cavern replaces Gritzy Desert (A10) for some reason.

Those are the only major errors, the rest of was just consistency fixes (like (Super Smash Bros. Melee) vs. (Melee) for example).

somealternateaccount commented:
@JackpotSHUnited It would be nice if you could squash the commits before we merge this, and I believe that's the only thing blocking this right now.

JackpotSHUnited commented:
I seriously have no idea how to use the "rebase" command or whatever it is I'm supposed to do.

bb010g commented:
What is the article not covering?

bb010g commented on projectm/pf/sound/strm/SONGS REPLACED.txt:
```diff
@@ -1,28 +1,29 @@
+A13 - Delfino Plaza                           ->  Defino's Secret
+A16 - Ground Theme 2 (Super Mario Bros.)      ->  Metal Mario (N64)
+A17 - Mario Bros.                             ->  Fountain of Dreams (Melee)
+A20 - Mario Circuit                           ->  Bowser's Castle ver. M
```
Inconsistent capitalization (see also L15).

jojmaht replied:
I won't validate the changes until this is fixed. I know it's a minute detail but still.

hhsnopek commented:
@JackpotSHUnited 
cmd: `git rebase -i HEAD~3`
change `pick` on the second and third commit to `fixup` or `f`, save and then `git log` (make sure you have the single commit), `git push origin patch-1 -f`

TheOriginalSINe commented:
Approved. It's just a text file change, so I can easily see everything that got changed.

bb010g commented:
Moved over to #287.
